src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -233,7 +240,10 @@ object ScalafixPlugin extends AutoPlugin {
 
   override def buildSettings: Seq[Def.Setting[_]] =
     Seq(
-      scalafixScalaBinaryVersion := "2.12"
+      scalafixScalaBinaryVersion := "2.12",
+      scalafixJGitCompletion := new JGitCompletion(
+        (ThisBuild / baseDirectory).value.toPath
+      )
```

Comment:
The explicit `ThisBuild` scope looks redundant here
```suggestion
      scalafixJGitCompletion := new JGitCompletion(baseDirectory.value.toPath)
```